data-gathering/serialize_data.py

```diff
@@ -0,0 +1,175 @@
+
+
+
+
+import json
+import urllib.parse
+
+# Define a dictionary or configuration file for each Wikipedia language
+discussion = "Discusión:"
+wikipedia_config = {
+    "es": {
+        "file_to_be_analysed": "query.csv",
+         "language": "es",
+         "utf_required": "utf-8",
+         "id_wikidata": 1,
+         "dimension": 1,
```

Comment:
Thank you for the reviews. I almost jumped out of my skin when I saw more than 3 comments but it pushed me to do more research. I've deleted the "serialize_data.py" file, thank you for that correction. 
Also concerning parameters from "dimension" to "discussion_size", I had to add all the parameters to not miss out on the important ones. It can be modified to meet the desired design pattern. I have made the changes suggested, kindly review them again while we await @SannitaSSJ